src/styles.css/cards.css

```diff
@@ -0,0 +1,74 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 32px 16px;
+  width: 200px;
+  font-family: Roboto, sans-serif;
```

Comment:
Move it to the `html` tag

Reply:
I do not understand. I set the size of the parent block. If I transfer to HTML, my block card will blur across the screen